new-lamassu-admin/src/App.js

```diff
@@ -91,12 +92,26 @@ const Main = () => {
     ? classes.contentWithSidebar
     : classes.contentWithoutSidebar
 
+  const transitionProps = {
+    direction: 'left'
+  }
+
   return (
     <div className={classes.root}>
       {!is404 && wizardTested && <Header tree={tree} />}
       <main className={classes.wrapper}>
         {sidebar && !is404 && wizardTested && (
-          <TitleSection title={parent.title}></TitleSection>
+          <Slide
+            {...transitionProps}
```

Comment:
why not just 
```suggestion
            direction="left"
```

Also this makes it so that every time we open this page the title slides to the left. But what I we add a new page after this one?
I accidentally found out this bug: every time we open operator info page, the title slides left. But when we change to a page on the left of this one, the title should slide right, but it still slides left, here's a video demonstrating the issue: https://streamable.com/14a6cc

Reply:
yeah, I talked to Rafa about this. It's a little hard to animate the title on this page because of how it's passed onto app.js. I think this is a fine workaround for now just to be consistent with the other pages.

Reply:
as for the other bug, I'll copy-paste your finding in the card, probably it's better to create a new card for this.